packages/framework-integration-tests/integration/provider-unaware/load/entities-are-correctly-reduced.load.ts

```diff
@@ -0,0 +1,77 @@
+import { ApolloClient } from 'apollo-client'
+import { NormalizedCacheObject } from 'apollo-cache-inmemory'
+import { internet, random } from 'faker'
+import gql from 'graphql-tag'
+
+import { waitForIt } from '../../helper/sleep'
+import { applicationUnderTest, scriptExecutor } from './setup'
+
+describe('Data consistency on entities', () => {
```

Comment:
This is the first load test: It sends more than 20,000 events and verifies that the entities (although we query ReadModels, they are a mapping one to one to entities) state has not been corrupted